### ISSUE TITLE
download_data.sh without flags does not error

### ISSUE THREAD

**Issue by paskino** [SyneRBI/SIRF-Exercises#152]
Running the download script without flags should prompt the usage text. However, I found the following which seems to suggest that everything went fine:

```

./scripts/download_data.sh 
Destination is "/home/sirfuser/devel/SIRF-Exercises/data"
Download location is "/home/sirfuser/devel/SIRF-Exercises/data"
creating data_path.py in /home/sirfuser/devel/SIRF-Exercises/lib/sirf_exercises/data_path.py
All done!
```

**Comment by KrisThielemans** [comment]
this might be not so obvious. I think it is intended to work like this, i.e. if no -p or -m, create folders, but don't download anything. Maybe we could handle this quite simply by letting it work as it does now but write a message

```
download_data.sh
WARNING: no options are added so data will not be downloaded.
We will just create a working directory. Use the -h option to find out more.

...
```

**Comment by KrisThielemans** [comment]
fixed in 0b089f671042ed74a089c2ab135ea6bd176f5e40
